src/MeasurementCodeInjector.php

```diff
@@ -70,58 +70,41 @@ public function injectEventTracking() {
         ?>
         <script>
             /**
-             * Keeps track of Ninja forms that have already been tracked
+             * Checks if the DOM event matches the Measurement Event or if any element in its path does
              *
-             * @type Array of elements
+             * @param e - DOM Event
+             * @param config - Measurement Event configuration
+             * @return {boolean}
              */
-            let ninjaFormsAddedNodes = [];
+            function pathHasMatch(e, config) {
+                let elemList = document.querySelectorAll(config.selector);
```

Comment:
Does this approach work if the target is a child element of the tracked element

e.g. tracking .add-to-cart

```
<button class="add-to-cart">
  <img class="icon" > Add to Cart
</button>
```

and the click lands on the icon image element

Reply:
Yes the pathHasMatch function checks for that